plugins/outputs/nebius_cloud_monitoring/nebius_cloud_monitoring.go

```diff
@@ -242,3 +242,15 @@ func init() {
 		return &NebiusCloudMonitoring{}
 	})
 }
+
+func replaceReservedTagNames(tagNames map[string]string) map[string]string {
+	newTags := make(map[string]string)
+	for tagName, tagValue := range tagNames {
+		if tagName == "name" {
```

Comment:
Is `name` the only reserved keyword?

Reply:
As I understand from support team - yes. There are direct message from support in linked issue.

Reply:
So, what about `ts` and `value`? They work correctly?

Reply:
Yes. I will attach screenshot how it looks like in Nebius. Right now it's working correctly. I just want to use suggestion about _name.

Reply:
@Hipska I think you are mixing up two things here. It's not that you cannot use `name` because it exists above the `labels` list, it's because it seems to be reserved in Nebius which creates a `name` label itself. Therefore there might be no relation to `ts` and `value` just because it also exists in the JSON... In any case, if we find more reserved labels later, we can still add them.

Reply:
It would clear up if @abrekhov would quickly test it, just to be sure.

Reply:
I hope that nobody will put ts and value as labels :) But if there will be cases - we will add it.

Reply:
But did you test if they give problems or not?